Python Tutorial Exercises(Week 5)/#2.py

```diff
@@ -0,0 +1,15 @@
+'''
+Take an input string from the command line/terminal, 
+and remove all vowels (a, e, i, o, u) from that string using list comprehension. 
+Use.lower() to convert the uppercase to lowercase
+Then see if the list vowels is in the inputs or not, if can find them in inputs list remove all.
+'''
+inputs = 'This is some input string'
+vowels = ('A', 'E', 'I', 'O', 'U')
+
+vowels = ''.join(vowels).lower()
```

Comment:
I thought you meant to get the the same outcome if the vowels's strings are uppercase letters. 
So the inputs is "ThIs Is sOmE InpUt StrIng"? Yep can do that.

Reply:
No the input can be anything since it is from the user on the terminal, so you should test this with different inputs. `"ThIs Is sOmE InpUt StrIng"` is just an example where the solution provided will have some short comings.